WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListItemMenuAdapter.kt

```diff
@@ -0,0 +1,78 @@
+package org.wordpress.android.ui.activitylog.list
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.content.res.AppCompatResources
+import org.wordpress.android.R
+import org.wordpress.android.ui.activitylog.list.ActivityLogListItem.SecondaryAction
+import org.wordpress.android.ui.activitylog.list.ActivityLogListItem.SecondaryAction.DOWNLOAD_BACKUP
+import org.wordpress.android.ui.activitylog.list.ActivityLogListItem.SecondaryAction.RESTORE
+import org.wordpress.android.util.ColorUtils.setImageResourceWithTint
+import org.wordpress.android.util.getColorResIdFromAttribute
+
+class ActivityLogListItemMenuAdapter(
+    context: Context
+) : BaseAdapter() {
+    private val inflater: LayoutInflater = LayoutInflater.from(context)
+    private val items: List<SecondaryAction> = SecondaryAction.values().toList()
+    override fun getCount(): Int {
+        return items.size
+    }
+
+    override fun getItem(position: Int): Any {
+        return items[position]
+    }
+
+    override fun getItemId(position: Int): Long {
+        return items[position].itemId
+    }
+
+    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+        var view = convertView
+        val holder: ActivityListItemMenuHolder
+        if (view == null) {
+            view = inflater.inflate(R.layout.activity_log_list_item_menu_item, parent, false)
+            holder = ActivityListItemMenuHolder(view!!)
```

Comment:
Warning (⚠️ ): This is not a warning per se, since it is guaranteed that the `!!` will never fail, but going forward and with more potential updates to it can create problems. Consider the following alternative:
```
val view: View
...
if (convertView == null) {
    view = inflater.inflate(R.layout.activity_log_list_item_menu_item, parent, false)
    ...
} else {
    view = convertView
    ...
}
```

Reply:
Good call out - in commit 2054d2b